### ISSUE TITLE
How taxes work (for foreigners)

### ISSUE THREAD

**Issue by toothfairy** [hyperoslo/playbook#46]
Only the wise man on a mountaintop knows, but try to explain in a nutshell what are taxes in Norway, and why the first June is a joke

**Comment by timkurvers** [comment]
You're thinking about the lie that is 'holiday money', @toothfairy? :+1:

**Comment by toothfairy** [comment]
@timkurvers absolutely!

**Comment by ohenrik** [comment]
I can maybe add a section about this late today :)

**Comment by SteveBenner** [comment]
As the newest addition to Hyper, this issue is of particular interest to me; in addition to Norway taxation, I will **also** have to figure out the USA tax situation, since it looks like I will be earning some income from remote work before I move to Oslo. This would be my first time reporting personal income significant enough that the IRS (the US federal ~~mafia~~ tax agency) would require I file my own tax return. I have proudly avoided contributing a single dollar to the American war machine this far, but as I don’t want to jeopardize my relocation to Norway, I will be doing all the necessary research into the US tax requirements I face in the near future.

Basically, any info you guys can provide would **especially** help me out since I already have a lot on my plate with the US side of things :)

**Comment by jgorset** [comment]
You won't need to pay taxes to Norway on your income before you get here, though, @SteveBenner. :smile:
